packages/story-editor/src/app/currentUser/currentUserProvider.js

```diff
@@ -33,7 +33,7 @@ function CurrentUserProvider({ children }) {
   } = useAPI();
 
   useEffect(() => {
-    if (!Object.keys(currentUser).length) {
+    if (getCurrentUser && !Object.keys(currentUser).length) {
```

Comment:
Would it make sense for `getCurrentUser` to default to a function that returns an empty object or `null` or something?

Reply:
You mean like this `() => Promise.resolve( {} );` . But why only for `getCurrentUser` ?

Reply:
Yeah. And that is a good question 🙂   Doesn't really make sense now that I think about it.

I was just wondering whether this one should be considered mandatory, but we don't use it for any critical stuff in the editor, so should be OK.